VideoLocker/src/main/java/org/edx/mobile/module/db/impl/IDatabaseImpl.java

```diff
@@ -228,6 +227,21 @@ public Boolean isVideoDownloadedInSection(String enrollmentId,
     }
 
     @Override
+    public List<Long> getDownloadedVideoDmIdsForSection(String enrollmentId, String chapter,
+                                                        String section , final DataCallback<List<Long>> callback ) {
```

Comment:
Also, we should probably create a query to just return the count, since that's really what we need here and it's more efficient as well.